### PR TITLE
End User Survey 2021

### PR DESCRIPTION
**Pull request by abuango**
PR adds the result of the 2021 ENd User survey

### CODE REVIEW

**Comment by abuango** [comment]
@kgamanji Kindly review

**Comment by amye** [comment]
Will you be adding raw data in here? (Scrubbed of PII *of course* 😁 )

**Comment by mkbhanda** [comment]
Q1: Did the survey not reach participants globally or is indeed CNCF having no participation in Asia, Africa, Australia  etc?
May want to update image to address this.
Q20: Recommend fixing typos in the user responses. Example, the first and second in the list.
Q26: Add some formatting for the 4 th response listed. The last response is revealing of tight balancing act.
Q3: reveals small companies do not have the time to participate and generally larger companies more involved.
Q4: revealing -- need to get the academic community more involved, some student hackathons, and analysis projects to start open source involvement early
Q5: NOT ACCESSIBLE, repeated error to view
Q8: legend on left incompletely visible
Q9: legend on left incompletely visible and the top two match -- PLEASE FIX

**Comment by abuango** [comment]
@amye Adding only the slides for now as per discussion with @kgamanji  

@mkbhanda Thank you for the feedback. There is participating globally, the survey was sent to all active End User members, the result reflects those who responded and filled the survey. I believe an action item will be to drive more participation in surveys, so it will be more representative. All other corrections suggested will be effected, appreciated.

**Comment by kgamanji** [comment]
The key takeaways and screenshots look great. I have observed that all screenshots have the "exit button" at the bottom-right. Can you remove that?

Otherwise LGTM

File: enduser/2021/README.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+The results are also [available on Google Slides](https://docs.google.com/presentation/d/1YO1n-N16ObskUQ54Zchn-fnAmcn715ZKMvs6_9plUPA/edit?usp=sharing)
+
+# Summary
+
+![summary](images/summary.jpg)
+
+## Takeaways and action items
+
+- The Pandemic has contributed to more participant in activities virtually
+- Member Organizations want to increase Open Source Contributions with Guidance from the CNCF
```
Comment (kgamanji) [inline enduser/2021/README.md]: Change "Member Organizations" to "Ens User Organizations". Not all End Users are members, some of them are supporters

File: enduser/2021/README.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+- Member Organizations see sharing & learning from peers as the most valuable part of their membership and would like to get more opportunities to do so.
```
Comment (kgamanji) [inline enduser/2021/README.md]: Same as above comment

**Comment by abuango** [comment]
@amye this can be merged, thanks.

**Pull request merged**